pkg/utils/utils.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/containers/image/pkg/sysregistries"
+)
+
+// CheckConfigFiles log messages if one or more files to set up buildah do not exist
+func CheckConfigFiles() error {
+	errMsg := ""
+	registriesConfPath := sysregistries.RegistriesConfPath(nil)
+	_, err := os.Stat(registriesConfPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			errMsg = fmt.Sprintf("Required file %s not found.\n", registriesConfPath)
+		} else {
+			errMsg = fmt.Sprintf("error checking %s, %s", registriesConfPath, err)
+		}
+	}
+
+	storageDefaultPath := "/etc/containers/storage.conf"
+	_, err = os.Stat(storageDefaultPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			errMsg = fmt.Sprintf("Required file %s not found.\n %s", storageDefaultPath, errMsg)
+		} else {
+			errMsg = fmt.Sprintf("error checking %s, %s.\n %s", storageDefaultPath, err, errMsg)
+		}
+	}
+	seccompDefaultPath := "/usr/share/containers/seccomp.json"
+	seccompOverridePath := "/etc/crio/seccomp.json"
+	_, errDefault := os.Stat(seccompDefaultPath)
+	_, errOverride := os.Stat(seccompOverridePath)
+	if errDefault != nil && errOverride != nil {
+		if os.IsNotExist(errDefault) && os.IsNotExist(errOverride) {
+			errMsg = fmt.Sprintf("Required file %s or %s not found.\n %s", seccompDefaultPath, seccompOverridePath, errMsg)
+		} else {
+			errMsg = fmt.Sprintf("error checking %s and %s, %s, %s.\n %s", seccompDefaultPath, seccompOverridePath, errDefault, errOverride, errMsg)
+		}
+	}
+
+	policyDefaultPath := "/etc/containers/policy.json"
+	_, err = os.Stat(policyDefaultPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			errMsg = fmt.Sprintf("Required file %s not found.\n %s", policyDefaultPath, errMsg)
+		} else {
+			errMsg = fmt.Sprintf("error checking %s, %s.\n %s", policyDefaultPath, err, errMsg)
+		}
+	}
```

Comment:
Why is existence of this file necessary before processing every single command, including things like `buildah version`?

Code paths that do need this file will eventually fail, so it seems to me that just making sure the error reporting/propagation works correctly should give the users all necessary information, e.g.
```console
$ ./buildah pull --signature-policy /foo/bar foo
error obtaining default signature policy: open /foo/bar: no such file or directory
```
That’s _already_ more useful than the text added above, which says “required file” and does not even say what it is for.  (Sure, the text added above can be easily improved, while “no such file or directory” might not quite as easy to augment.)